server/models/userProfile.js

```diff
@@ -9,7 +9,18 @@ const ProfileSchema = new Schema({
   picture: String,
   errandsCompleted: Number,
   location: String,
-});
+  // userCoordinates: {
+  //   type: {
+  //     type: String,
+  //     default: 'Point',
```

Comment:
Sorry Oje, I was debating on whether or not we should use GeoJSON in the user schema since it's a location-based app. Commented out that part for now.